src/mn_internal.h

```diff
@@ -102,7 +102,7 @@ void MN_DrawEnemy(void);
 #define S_HILITE      0x00000001 // Cursor is sitting on this item
 #define S_SELECT      0x00000002 // We're changing this item
 #define S_TITLE       0x00000004 // Title item
-
+#define S_FUNC        0x00000008 // Call var.func() on MENU_ENTER
```

Comment:
This uses one of the only flags left here.

Reply:
No problem, we can add `flags2` or make `flags` 64-bit.